CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+- The `import` subcommand gained the `--listen,l` option to receive input
```

Comment:
```suggestion
- The `import` command gained the `--listen,l` option to receive input
```

I would consider subcommands an implementation detail. It's all the same for the user.